README.md

```diff
@@ -214,6 +214,10 @@ The example above contains an erroneous 'x' at the end, so it does not validate:
 > It will be removed (`https://example.com/01/00860080001300`) in the validation trace. That's because the
 > Digital Link Grammar file doesn't support the custom paths.
 
+> Warning : The isValid method also checks if the identifier has a valid check digit. If it isn't the case, it will 
+> return false. However, the getValidationTrace won't show anny error since it doesn't take into account the check digit
```

Comment:
`getValidationTrace` might not be used much, but I would also add this gotcha to the docs for that too.